src/libs/workspace-utils.ts

```diff
@@ -107,6 +108,22 @@ export interface WorkspacePolicy {
   additionalData: { [key: string]: string }[];
 }
 
+export const getPolicyShortDescription = (policy: WorkspacePolicy): string => {
+  return Utils.cond(
```

Comment:
I need to change this so that it combines policies with the same name (we don't want to show multiple group-constraint or region-constraint entries, for example).